src/app/modules/products/components/product-list/product-list.component.html

```diff
@@ -1,14 +1,5 @@
-<section class="product-list-container">
-    <div class="product-list">
-        <div class="product-list-items-container">
-            <div class="product-card-featured-container" *ngFor="let product of featuredProducts$ | async">
-                <app-featured-product-card [product]="product"></app-featured-product-card>
-            </div>
-        </div>
-        <div class="product-list-items-container">
-            <div class="product-card-container" *ngFor="let product of products$ | async">
-                <app-product-card [product]="product"></app-product-card>
-            </div>
-        </div>
+<div class="product-list-items-container">
```

Comment:
i think in this case it would be preferable, to leave the "product-list-items-container" in product-page.

This element does not need to know how its container is positioning it.

Reply:
All good